content/blog/2025-12-04-invenioils-product/contents.lr

```diff
@@ -0,0 +1,70 @@
+title: Repositories and libraries: Invenio features InvenioILS!
+---
+author: Karolina Przerwa
+---
+pub_date: 2025-12-04
+---
+team: CERN
+---
+body:
+
+For years, the Invenio community has been known for powerful open-source repository solutions - especially InvenioRDM. But many institutions need more than a repository: they need a modern, flexible platform that also supports collections, loans, acquisitions, and patron services.
+
+**InvenioILS brings these capabilities into the Invenio family**, offering a fully open, customizable **Integrated Library System** built on the same modular framework as InvenioRDM.
+
+
+## A modern, feature-rich ILS (Integrated Library System)
+
+InvenioILS delivers the essential tools libraries rely on:
+
+- **Circulation & patron services**: loans, renewals, requests, notifications, and configurable policies.
+- **Cataloguing & metadata**: flexible JSONSchema-based records, holdings, items, and fast discovery powered by Elasticsearch/OpenSearch.
+- **Acquisitions & ILL**: vendor management, orders, budgets, document delivery, and interlibrary loan workflows.
+- **Clean, responsive UI**: a modern interface designed for both librarians and patrons.
+
+
+## Flexible and extensible by design
+
+InvenioILS is highly modular and, like InvenioRDM, based on InvenioFramework. It becomes easier to share the expertise between these two products.
+Institutions can customize metadata, circulation rules, UI components, and system behaviour - or integrate with external services through the REST API. This flexibility allows libraries to adopt InvenioILS “as is” or tailor it into a specialized solution that fits their workflows and identity.
+
+## CERN: A Showcase Installation
+
+An example of an advance deployment of InvenioILS is at **CERN** - [CERN Library Catalogue](https://catalogue.library.cern), where the platform powers daily library operations and serves thousands of users.
+
+![](clc.png)
+
+### Smooth SSO integration
+
+InvenioILS connects seamlessly to CERN’s institutional Single Sign-On, providing automatic user provisioning, role synchronization, and a unified login experience across CERN services.
+
+### Bulk e-book importer
+
+CERN uses an automated bulk importer to process large vendor MARC packages. Metadata is transformed, validated, and ingested with minimal staff intervention - turning what once took days into an efficient, scalable workflow.
+
+### Mobile self-checkout & campus navigation
+
+CERN’s library offers mobile self-checkout, enabling patrons to borrow items directly from their phones. Through InvenioILS APIs, the system also integrates with CERN’s internal navigation app, guiding users to shelf locations and showing real-time item availability.
+
+![](maps.png)
+
+![](self-checkout.png)
```

Comment:
Does it make sense to group these in a section related to potential or example customizations/integrations?
It is unclear when reading if such features are available out-of-the-box or not